README.md

```diff
@@ -40,6 +40,21 @@ make deploy
 
 This will leave changes on disk in your `config` directory, take care not to commit them.
 
+### Deploying webhooks / Installing Cert-Manager
+
+This Operator supports [defaulting and validating webhooks](https://book.kubebuilder.io/reference/webhook-overview.html),
+which ensure the validity of any `EtcdCluster` and `EtcdPeer` custom resources that you create.
+
+Running `make deploy`  will configure these webhooks.
+For any CREATE or UPDATE operation on any `EtcdCluster` or `EtcdPeer` custom resource,
+the Kubernetes API server initiate HTTPS webhook requests to the `controller-manager`,
+so that it can perform defaulting and validation of the resource,
+before it is stored by the Kubernetes API server.
+
+These HTTP webhook requests, require SSL certificates to be configured for each of webhook request type,
+and the easiest way to set this up, is to install [Cert-Manager](https://book.kubebuilder.io/cronjob-tutorial/running-webhook.html#cert-manager),
+before you deploy the Operator.
```

Comment:
maybe specify that these are self-signed certs and will not require any external dependency on any CA (assuming that's right)?